cfgov/jinja2/v1/_includes/organisms/sidebar-breakout.html

```diff
@@ -0,0 +1,45 @@
+{# ==========================================================================
+
+   sidebar_breakout.render()
+
+   ==========================================================================
+
+   Description:
+
+   Creates Sidebar Breakout markup when given:
+
+   settings:               An object used to customize the markup.
+
+   settings.heading:       A string containing heading text.
+
+   settings.image:         A string containing an image url.
+
+   settings.intro.heading: A string containing heading text.
+
+   settings.intro.body:    A string containing body text.
+
+   ========================================================================== #}
+
+{% macro render(settings) %}
+<div class="o-sidebar-breakout">
+    <h3 class="o-sidebar-breakout_heading">
+        {{ settings.heading }}
+    </h3>
+    {% if settings.image %}
+    <div class="o-sidebar-breakout_column">
+        <div class="u-flexible-container">
+            <div class="u-flexible-container_inner"
+                 style="background-image:url( {{settings.image}} );">
+            </div>
+        </div>
+    </div>
+    {% endif %}
+    <div class="o-sidebar-breakout_column">
+        <h4 class="o-sidebar-breakout_intro-heading">
+            {{ settings.intro.heading }}
+        </h4>
+        <p>{{ settings.intro.body }}</p>
+    </div>
+    {# Add Related-Posts molecule once #}
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/fccbc45d997b7e141e032541bfac0cda5779311b